target/linux/ath79/image/Makefile

```diff
@@ -66,6 +74,28 @@ define Device/Default
 	append-rootfs | pad-rootfs | append-metadata | check-size $$$$(IMAGE_SIZE)
 endef
 
+define Device/seama
+$(Device/Default)
+  LOADER_TYPE := bin
+  KERNEL := kernel-bin | append-dtb | relocate-kernel | lzma
+  KERNEL_INITRAMFS := kernel-bin | append-dtb | relocate-kernel | lzma | seama
+  KERNEL_INITRAMFS_SUFFIX = $$(KERNEL_SUFFIX).seama
+  IMAGES := sysupgrade.bin factory.bin
```

Comment:
`IMAGES += factory.bin` is enough. sysupgrade.bin is already added in Makefile:define Device/Default.

Reply:
OK